lib/msgr/version.rb

```diff
@@ -4,7 +4,7 @@ module Msgr
   module VERSION
     MAJOR = 0
     MINOR = 15
-    PATCH = 2
```

Comment:
Don't touch the version number in feature PRs. please. :)

Reply:
Yes, please.

At least it would be a minor bump to `0.16.0`.

@franzliedke How about actually jumping to `1.0`?

Reply:
Sorry, this shouldn't go here. It was just for testing.